src/chrome/content/rules/2o7.net.xml

```diff
@@ -1,22 +1,18 @@
 <!--
 	For other Adobe coverage, see Adobe.xml.
 
-
-	Fully covered domains:
-
-		- *.sbx1.2o7.net
-
-
-	If we are going to include ads and tracking info, let's at least do it by https.
-
 -->
 <ruleset name="2o7.net">
 
 	<target host="*.2o7.net" />
+		<test url="http://122.2o7.net/optout.html" />
+		<test url="http://www.112.2o7.net/optout.html" />
+		<test url="http://independent.122.2o7.net/b/ss/indepdev/1/H.27.5--NS/0" />
+		<test url="http://cewecolor.112.2o7.net/optout.html" />
+		<test url="http://truckstore.122.2o7.net/optout.html" />
 
 
-	<securecookie host="^(?:.*\.)?2o7\.net$" name=".+" />
-
+	<securecookie host=".+" name=".+" />
 
 	<rule from="^http://([^/:@\.]+)(\.1[12]2|\.sbx1)?\.2o7\.net/"
 		to="https://$1$2.2o7.net/" />
```

Comment:
Can we simplify this somehow or is there a known issue?

Reply:
I'm not sure. See my edit for a better but still safe solution.